system-tests/universe/test-universe.js

```diff
@@ -87,5 +138,51 @@ describe("Universe", function() {
     cy.get(".modal").contains("Review and Deploy").click();
     cy.get(".modal").contains("Deploy").click();
     cy.get(".modal").contains("Go To Service").click();
+
+    // Go to the root services page
+    cy.get(".breadcrumbs").contains("Services").click();
+
+    // Check that it appears in the service list
+    cy
+      .get(".page-body-content table", {
+        timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT
+      })
+      .contains(serviceName, { timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT })
+      .should("exist");
+  });
+
+  it("deletes an already installed package", function() {
+    const packageName = "bitbucket";
+    const serviceName = `${Cypress.env("TEST_UUID")}-${packageName}`;
+
+    cy.visitUrl("services/overview");
+
+    // Click on the name of the package to delete
+    cy
+      .get(".page-body-content table")
+      .getTableRowThatContains(serviceName)
+      .get("a.table-cell-link-primary")
+      .contains(serviceName)
+      .click();
+
+    // Click delete in the dropdown
+    cy
+      .get(".page-header-actions .dropdown")
+      .click()
+      .get(".dropdown-menu-items")
+      .contains("Delete")
+      .click();
+
+    // Confirm the deletion
+    cy.get(".modal.confirm-modal input").type(serviceName);
+    cy.get(".modal.confirm-modal").contains("Delete").click();
+
+    // Give it a bit of time to dissapear from service list
```

Comment:
This is not necessary since Cypress will retry the command until it resolves up to `defaultCommandTimeout`. More info: https://docs.cypress.io/docs/making-assertions#section-resolving-assertions